### PR TITLE
Release 2.0.0 fixes

### PR DESCRIPTION
**Pull request by pconesa**

### CODE REVIEW

**Comment by pconesa** [comment]
Closes #1935

File: pyworkflow/apps/pw_schedule_run.py [hunk 1]
```diff
@@ -192,6 +192,7 @@ def _getProtocolFromPointer(pointer):
 
         _log("Launching the protocol >>>>")
         log.close()
+        time.sleep(1)
```
Comment (dmaluenda) [inline pyworkflow/apps/pw_schedule_run.py]: Is it a need? or a debug?

File: pyworkflow/install/script.py [hunk 2]
```diff
@@ -196,22 +196,22 @@ def removeByPattern(pattern):
...
-         default=False)
+         default=True)
 
     sqlite = env.addLibrary(
         'sqlite3',
         tar='SQLite-1a584e49.tgz',
         flags=['CPPFLAGS=-w',
                'CFLAGS=-DSQLITE_ENABLE_UPDATE_DELETE_LIMIT=1'],
-        default=False)
+        default=True)
```
Comment (dmaluenda) [inline pyworkflow/install/script.py]: We have 'sqlite' as a 'python' dependency, does it make sense? I agree with this default

File: pyworkflow/apps/pw_schedule_run.py (hunk 1 above)
Comment (fonsecareyna82) [reply pyworkflow/apps/pw_schedule_run.py]: Thanks David, that instruction is unnecessary

**Comment by pconesa** [comment]
Closes #1936

**Comment by dmaluenda** [comment]
Which procedure we will follow here? I mean, this PR is becoming huge and not easy to review since it's recovering different scopes...

**Comment by pconesa** [comment]
Someone must review it, I've created and cannot approve my own PR. Please anyone there could review to your best of the knowledge and if Ok approve it. This is the branch tested in buildbot, so no major breaks.

**Comment by pconesa** [comment]
@delarosatrevin , @dmaluenda is it ok to merge this.... please have a look to what you are familiar with.

File: pyworkflow/em/protocol/protocol_align_movies.py [hunk 3]
```diff
@@ -269,23 +269,21 @@ def _validate(self):
...
+        firstItem = inputMovies.getFirstItem()
+
+        firstFrame, lastFrame, _ = inputMovies.getFramesRange()
         if lastFrame == 0:
             # Although getFirstItem is not recommended in general, here it is
-            # used olny once, for validation purposes, so performance
-            # problems should not appear.
-            frames = self.inputMovies.get().getFirstItem().getNumberOfFrames()
+            # used only once, for validation purposes, so performance
+            # problems not should appear.
```
Comment (pconesa) [inline pyworkflow/em/protocol/protocol_align_movies.py]: should not was correct, ;-)

Comment (delarosatrevin) [reply pyworkflow/em/protocol/protocol_align_movies.py]: Ops, sorry, I think I messed up the comment and then "fixed", but not entirely, I will correct it.

**Pull request merged**